src/core/main/ts/dom/Position.ts

```diff
@@ -34,6 +34,18 @@ const getTableCaptionDeltaY = function (elm) {
   }
 };
 
+const hasChild = function (elm, child) {
+  if (elm && elm.children) {
+    for (let i = 0; i < elm.children.length; i++) {
+      if (elm.children[i] === child) {
+        return true;
```

Comment:
Isn't this identical to `child.parentNode === elem`?

Reply:
That would be the case only if elements are HTMLElement instances. Angular and other dynamic DOM frameworks inject elements which from what I found out are not properly represented in parent-child node relationships. That's why with every level I needed to check all children to see if maybe we are about to skip the root element we are looking for.

The rought DOM looks like this (taken from Chrome):

```
<div _ngcontent-c24="" class="col-sm-12">
  <html-editor _ngcontent-c24="" elementid="question-feedback" _nghost-c37="">
    <div _ngcontent-c37="" class="menu-holder" id="question-feedback-menu">
	  <!-- toolbar -->
	</div>
	<div _ngcontent-c37="" class="html-editor mce-content-body mce-edit-focus" id="question-feedback" spellcheck="false" contenteditable="true"><p><br data-mce-bogus="1"></p></div>
    <input type="hidden" name="question-feedback">
  </html-editor>
</div>
```

Hope it clarifies the reason for scanning for children each level (parentNode is already used but not enough).

Reply:
Oh dear. That's a bit silly. OK, we'll keep the children search instead of the parent check.

I might change it to use `Array.prototype.indexOf.call()` which we do in other places for stuff like this. It should be in soon.